src/components/BookList.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable */ 
+import React from 'react';
+import Book from './Book';
+
+const dummyData = [{
+  title: 'Dunes',
+  author: 'Frank Herbert',
+  genre: 'Action',
+},
+{
+  title: 'Alchemist',
+  author: 'Paulo Cohelo',
+  genre: 'Fiction',
+},
+];
+const BookList = () => (
+  <div>
+    {dummyData.map((book) => (
+      <Book title={book.title} genre={book.genre} author={book.author} />
+    ))}
+  </div>
```

Comment:
- [ ] Please as it is a requirement for this project, the form should be rendered in the `BookList` component. Please kindly import `InputBook.js` component and render it here.